main.go

```diff
@@ -41,7 +42,7 @@ func main() {
 	// Images.xcassets/AppIcon.appiconset/Contents.json
 	dir, err := ioutil.TempDir("", "go2xcode")
 	if err != nil {
-		panic(err)
+		log.Fatalln(err)
```

Comment:
Why do you prefer `log.Fatal` over `log.Fatalln`? I know they have the exact same behavior because log package always appends a newline after the call, but it just seems more consistent to do `log.Fatalln`. That way, if you change it to `fmt.Println`, the output remains the same.